standard/classes.md

```diff
@@ -1622,12 +1622,20 @@ member_name
 
 method_body
     : block
+    | '=>' null_conditional_invocation_expression ';'
     | '=>' expression ';'
     | ';'
     ;
 ```
 
-*unsafe_modifier* ([§23.2](unsafe-code.md#232-unsafe-contexts)) is only available in unsafe code ([§23](unsafe-code.md#23-unsafe-code)).
+Grammar notes:
+
+- *unsafe_modifier* ([§23.2](unsafe-code.md#232-unsafe-contexts)) is only available in unsafe code ([§23](unsafe-code.md#23-unsafe-code)).
+
```

Comment:
Unnecessary blank line here? The one before the previous bullet may well be unnecessary too.

Reply:
Hmmm... I'll admit to thinking that the blank line was optional in this case – sometimes you need a blank line to make a break and sometimes MD will infer the break, but in the latter case I thought you could be explicit and I tend to put them in or not depending on readability of the raw MD.

Just did a quick scan and in §15.3.1 I see lists with and without spaces and at least GitHub is rendering the spacing differently. So this problem, if it is one, could be all over the place...

Reply:
I don't think it's an actual problem, just a personal preference to make it clear that it's a single list. I'll resolve now as I don't want this to get in the way of the PR.